### PR TITLE
Swift Client generation v1

### PR DESCRIPTION
**Pull request by PaulTaykalo**

### CODE REVIEW

File: codegen/swift/swift-entities/src/main/kotlin/com/stanfy/helium/handler/codegen/swift/entity/entities/SwiftEntitiesGenerator.kt [hunk 1]
```diff
@@ -25,114 +23,48 @@ class SwiftEntitiesGeneratorImpl : SwiftEntitiesGenerator {
...
         .map { sequence ->
-          swiftType(sequence, typesRegistry)
+          typesRegistry.registerSwiftType(sequence)
         }
 
 
     return enums + messages + sequences
   }
 
-  fun propertyName(fieldName: String): String {
```
Comment (PaulTaykalo) [inline codegen/swift/swift-entities/src/main/kotlin/com/stanfy/helium/handler/codegen/swift/entity/entities/SwiftEntitiesGenerator.kt]: Moved to the types registry

File: helium/src/main/groovy/com/stanfy/helium/internal/utils/Names.java [hunk 2]
```diff
@@ -41,7 +41,7 @@ public static String prettifiedName(final String s) {
     StringBuilder result = new StringBuilder(s);
     for (int i = 0; i < result.length(); i++) {
       boolean shouldCap = false;
-      while (i < result.length() && (result.charAt(i) == '_' || result.charAt(i) == '-')) {
+      while (i < result.length() && (result.charAt(i) == '_' || result.charAt(i) == '-' || result.charAt(i) == ' ')) {
```
Comment (roman-mazur) [inline helium/src/main/groovy/com/stanfy/helium/internal/utils/Names.java]: - Add it to the method test please.
- We usually do something like `prettifiedName(canonicalName(s))`, where `canonicalName handles spaces`
- Also,  `Type`s and `Service`s provide (IIRC) `getCanonicalName()` method

**Comment by roman-mazur** [comment]
@PaulTaykalo Any plans on this? You think it can be merged to master?

**Comment by PaulTaykalo** [comment]
@roman-mazur unless you want to have better coverage.
It's working pretty smooth, so feel free to merge

**Pull request merged**